index.html

```diff
@@ -64,15 +66,15 @@ <h1>Bikes for Refugees</h1>
               Providing donated bikes and accessories to refugees and asylum
               seekers in Scotland.
             </p>
-            <button>Donate a bike today</button>
-            <button>Volunteer</button>
+            <button class="firstBS">Donate a bike today</button>
+            <button class="secondBS">Volunteer</button>
           </div>
 
           <div>
             <h2 class="heading-underline">Learn more</h2>
 
-            <div>
-              <div class="article">
+            <section>
+              <article>
```

Comment:
It would be useful to use a class also here, so that in the CSS you can define the ratio of flexbox.
Try to change here the `flex` attribute on children and see what happens:
https://www.w3schools.com/cssref/tryit.php?filename=trycss3_flex

if you use the CSS directly on article element , could affect also other `article` element, while we want apply this change only to this article element and the next one